<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal Processing</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Trabajos Unal/Teoria de Señales/TKinter/unal.ico</normaloff>../../Trabajos Unal/Teoria de Señales/TKinter/unal.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonWGN">
      <property name="text">
       <string>WGN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Draw Graph</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MplWidget" name="MplWidget" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport_Dummy">
     <property name="title">
      <string>Import Dummy</string>
     </property>
     <addaction name="actionSin_60Hz"/>
    </widget>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="menuImport_Dummy"/>
   </widget>
   <widget class="QMenu" name="menuModulator">
    <property name="title">
     <string>Modulator</string>
    </property>
    <widget class="QMenu" name="menuRFA">
     <property name="title">
      <string>RFA</string>
     </property>
     <addaction name="actionAM"/>
     <addaction name="actionFM"/>
    </widget>
    <addaction name="menuRFA"/>
    <addaction name="actionRFD"/>
    <addaction name="actionBBA"/>
    <addaction name="actionBBD"/>
   </widget>
   <widget class="QMenu" name="menuDemodulator">
    <property name="title">
     <string>Demodulator</string>
    </property>
    <addaction name="actionRFA_2"/>
    <addaction name="actionRFD_2"/>
    <addaction name="actionBBA_2"/>
    <addaction name="actionBBD_2"/>
   </widget>
   <widget class="QMenu" name="menuWGN">
    <property name="title">
     <string>Other</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionManual"/>
   </widget>
   <addaction name="menuHello"/>
   <addaction name="menuModulator"/>
   <addaction name="menuDemodulator"/>
   <addaction name="menuWGN"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import Signal</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export Signal</string>
   </property>
  </action>
  <action name="actionRFD">
   <property name="text">
    <string>RFD</string>
   </property>
  </action>
  <action name="actionAM">
   <property name="text">
    <string>AM</string>
   </property>
  </action>
  <action name="actionFM">
   <property name="text">
    <string>FM</string>
   </property>
  </action>
  <action name="actionBBA">
   <property name="text">
    <string>BBA</string>
   </property>
  </action>
  <action name="actionBBD">
   <property name="text">
    <string>BBD</string>
   </property>
  </action>
  <action name="actionRFA_2">
   <property name="text">
    <string>RFA</string>
   </property>
  </action>
  <action name="actionRFD_2">
   <property name="text">
    <string>RFD</string>
   </property>
  </action>
  <action name="actionBBA_2">
   <property name="text">
    <string>BBA</string>
   </property>
  </action>
  <action name="actionBBD_2">
   <property name="text">
    <string>BBD</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionSin_60Hz">
   <property name="text">
    <string>Sin 60Hz</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
